tardis/model/parse_input.py

```diff
@@ -152,4 +152,15 @@ def parse_abundance_section(config, atom_data, geometry):
     if atom_data is not None:
         elemental_mass = atom_data.atom_data.mass
 
+    assert (
+        abundance < 0.0
+    ).sum().sum() == 0, (
```

Comment:
I'm a bit confused about what this .sum().sum() == 0 is doing. Can you describe that operation?

Reply:
It essentially sums up all ``True`` (=1) and ``False`` (=0) values. Since all values in the expression before that should be ``False``, the sum should evaluate to 0. A single ``True`` value would mean the sum is larger than 0. The double sum is just to cover both axes of the DataFrame. As far as I know there is now built-in method to do this in pandas. A simple ``assert (abundance >=0).all()`` doesn't seem to work with DataFrames